UnitsNet/GeneratedCode/Quantities/Capacitance.NetFramework.g.cs

```diff
@@ -68,6 +79,32 @@ public Capacitance(double numericValue, CapacitanceUnit unit)
             _unit = unit;
         }
 
+        /// <summary>
+        /// Creates an instance of the quantity with the given numeric value in units compatible with the given <see cref="UnitSystem"/>.
+        /// </summary>
+        /// <param name="numericValue">The numeric value  to contruct this quantity with.</param>
+        /// <param name="unitSystem">The unit system to create the quantity with.</param>
+        public Capacitance(double numericValue, UnitSystem unitSystem)
```

Comment:
Already mentioned, but I think we should have only one ctor for either UnitSystem or BaseUnits. Not both.

Reply:
I think both are useful. I wanted to keep them as two classes for separation of concerns and single responsibility. UnitSystems are defined with BaseUnits, but BaseUnits do not make up a UnitSystem. You could do something like this for example when you want the SI prefixes:
```csharp
    public class SIUnitSystem : UnitSystem
    {
        private static readonly BaseUnits SIBaseUnits = new BaseUnits( LengthUnit.Meter, MassUnit.Kilogram, DurationUnit.Second,
            ElectricCurrentUnit.Ampere, TemperatureUnit.Kelvin, AmountOfSubstanceUnit.Mole, LuminousIntensityUnit.Candela );

        /// <summary>
        /// Gets the SI unit system.
        /// </summary>
        public static SIUnitSystem Instance { get; } = new SIUnitSystem();

        private SIUnitSystem() : base( SIBaseUnits )
        {
        }

        public const double KiloFactor = 1e3;
    }
```

Reply:
@angularsen